packages/wmr/src/lib/plugins.js

```diff
@@ -38,12 +38,13 @@ export function getPlugins(options) {
 	if (split === -1) split = plugins.length;
 
 	const production = mode === 'build';
+	const mergedAssets = new Set();
```

Comment:
This file tracks assets that aren't relevant to the final build, for instance sass-plugin inlining them